src/layers/medWidgets/process/arithmetic_operation/medAbstractArithmeticOperationProcessPresenter.cpp

```diff
@@ -72,17 +72,31 @@ medViewContainerSplitter *medAbstractArithmeticOperationProcessPresenter::buildV
     medViewContainer * outputContainer = inputContainer1->splitVertically();
     medViewContainer* inputContainer2 = inputContainer1->splitHorizontally();
 
-    inputContainer1->setDefaultWidget(new QLabel("INPUT 1"));
+    auto viewLayout1 = static_cast<QVBoxLayout*>(inputContainer1->defaultWidget()->layout());
```

Comment:
In a fix container that you can't remove (for instance in Filtering or Registration), you now can open a data from the buttons in the view or the Tools button. We still can add a label on the fix view, as done here. It will be displayed on top of the view buttons